### ISSUE TITLE
Configurable order summary columns

### ISSUE THREAD

**Issue by kentbower** [kbower/tickettest1#154]
_Imported from trac ticket 154.
Created by **rneville**
Opened in trac: May 13, 2010, 9:05:01 AM
Last modified in trac: Jul 02, 2010, 4:27:23 PM_
***
_by rneville:_
Order summary column in the Orders object needs to be configurable as to which fields are put together to form it (ve_cd and vsn being an example)

**Comment by kentbower** [comment]
_**rneville** trac comment on May 13, 2010, 2:05:29 PM:_

accepted

**Comment by kentbower** [comment]
_**kbower** trac comment on Jul 02, 2010, 4:27:23 PM:_

(In [[624]](../commit/168a0c6cdfba1c8dd1b4d6a37146ffcee2a8cf71)) Configurable order summary (fixes #154)

New `SystemParameter` `'ordersummary_fields'` added, default to `"%s,description"`, which describes how the order summary is presented.
Changing this to 
```
%s (%s),description,vendor
```would result in "Description (VND)"-type format for the products in the order summary.

**Comment by kentbower** [comment]
_**kbower** trac status change on Jul 02, 2010, 4:27:23 PM:_ **closed**
